### PR TITLE
Increase package utilities test coverage

### PR DESCRIPTION
**Pull request by pjlee11**
Resolves #224 

Follows up on https://github.com/BBC-News/psammead/pull/224 is dependant on https://github.com/BBC-News/psammead/pull/258.

Add tests for the package utilities using the newly created method in https://github.com/BBC-News/psammead/pull/258

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

To test this run `npm install && npm run build && npm run test` ensuring there are no errors. The test coverage for the packages `gel-foundations`, `psammead-assets` and `psammead-styles` should all change from `0%` to `100%`.

### CODE REVIEW

**Comment by dr3** [comment]
Only 42 commits? You could do with a boost ;)

File: packages/utilities/gel-foundations/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 | Version | Description |
 |---------|-------------|
+| 0.1.4   | [PR#224](https://github.com/BBC-News/psammead/pull/224) Add tests for the exported values, coverage 100% :tada: |
```
Comment (bcmn) [inline packages/utilities/gel-foundations/CHANGELOG.md:5]: What's happened to changelog info for `0.1.2` & `0.1.3`? 😬 The package.json seems to imply these versions existed.

I know this isn't your PR, just intriguing.

**Review by bcmn — APPROVED** [review]

**Review by dr3 — APPROVED** [review]

File: packages/utilities/gel-foundations/CHANGELOG.md (hunk 1 above)
Comment (pjlee11) [reply packages/utilities/gel-foundations/CHANGELOG.md:5]: Using git blame it seems this happened - https://github.com/bbc/psammead/commit/09345853fb76a483439691c35256c1dc2fbb1946 
🤐

Comment (pjlee11) [reply packages/utilities/gel-foundations/CHANGELOG.md:5]: For the wider context it was as part of this PR https://github.com/bbc/psammead/pull/212/files

**Comment by jamesbrumpton** [comment]
LGTM 👍

**Pull request merged**